README.md

```diff
@@ -99,6 +99,29 @@ For an example of identifying users in HTTP handlers, see [demo/http_context.cr]
 
 [Learn more about context data in Honeybadger](https://docs.honeybadger.io/guides/errors/#context-data)
 
+### Sending Events to Honeybadger Insights
+
+You can send custom events to [Honeybadger Insights](https://docs.honeybadger.io/guides/insights/) to track important business metrics and user actions in your application:
+
+```crystal
+# Send a simple event
+Honeybadger.event(name: "user.signup")
+
+# Send an event with properties
+Honeybadger.event(
+  name: "order.completed",
+  total: 99.99,
+  items: ["book", "shirt"],
+  user_id: 123
+)
+```
```

Comment:
@stympy I think this is incomplete -- these events as documented won't ever get sent.

Reply:
It works as expected for me: https://gist.github.com/stympy/2f2631cac90384a9b0bcfbd621c348f0